avaframe/runOperational.py

```diff
@@ -86,16 +88,16 @@ def runOperational(avalancheDir=''):
     # Collect results/plots/report  to a single directory
     # make simple plots (com1DFA, com2AB)
     # peak file plot
+    modName = 'com1DFA'
```

Comment:
I think these lines are not required anymore since the plotDict is passed from com1DFA